source/common/config/filter_json.cc

```diff
@@ -164,8 +164,10 @@ void FilterJson::translateHttpConnectionManager(
         "{\"deprecated_v1\": true, \"value\": " + json_filter->getObject("config")->asJsonString() +
         "}";
 
+    Protobuf::util::JsonParseOptions options;
```

Comment:
Drive by comment, can we potentially wrap this in a common utility function? Somewhere in common/protobuf/utility.h?

Reply:
Done. There is already a loadFromJson() function there, so just invoke it instead. Note that it throws an exception in case of error instead of returning the status code, hence there are some minor changes.